test/unit/core/algorithm/pipeable_config_element_test.cpp

```diff
@@ -79,25 +79,25 @@ TEST(pipeable_config_element, element_with_configuration)
     // lvalue | lvalue
     {
         auto cfg = b2 | tmp;
-        EXPECT_TRUE((std::is_same_v<decltype(cfg), seqan3::configuration<bar, bax>>));
+        EXPECT_TRUE((std::is_same_v<decltype(cfg), seqan3::configuration<bax, bar>>));
```

Comment:
Is there a reason why you changes the order?

Reply:
Well, before the order depended on which overload of the |-operator was invoked because for symmetry reasons we often had: `operator|(cfg_elem lhs, configuration rhs) { return rhs | lhs }`.
Now the order is always maintained, i.e. elements from lhs always come before elements of rhs.